x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/utils/promote_featured_integrations.ts

```diff
@@ -27,9 +27,9 @@ export function _promoteFeaturedIntegrations(
   );
 
   // now return the integrations in the order they are defined in
-  const orderedFeaturedIntegrations = featuredIntegrationNames
-    .map((integrationName) => featuredIntegrations.find(({ name }) => name === integrationName))
-    .filter((v) => v) as PackageListGridProps['list'];
+  const orderedFeaturedIntegrations = featuredIntegrations.filter(({ name }) =>
+    featuredIntegrationNames.includes(name)
+  );
 
```

Comment:
Before this change, a package with multiple policy templates appeared only once and filtered out the rest.

By using .filter() instead of .find(), this code will include all elements with matching names in the orderedFeaturedIntegrations array, thus ensuring that all packages with the same name are included in the final result.